plugin-server/src/cdp/templates/_destinations/email/email.template.ts

```diff
@@ -27,7 +27,7 @@ export const template: HogFunctionTemplate = {
             required: true,
             default: {
                 to: {
-                    email: '{person.properties.email}',
+                    email: '{{ person.properties.email }}',
```

Comment:
I think we need to also add something to the schema to indicate the templating language. We could change the `templating` param to be `false | true | 'hog' | 'liquid'` perhaps...

Reply:
i think this can be follow up though